src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -88,6 +88,10 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler,
                                  rpc::GetAllResourceUsageReply *reply,
                                  rpc::SendReplyCallback send_reply_callback) override;
 
+  void HandleRequestClusterResources(rpc::RequestClusterResourcesRequest request,
```

Comment:
This will be part of the GcsAutoscalerStateManager in the PR https://github.com/ray-project/ray/pull/35596